backend/Infrastructure/Repositories/OrganisationDbRepository.cs

```diff
@@ -0,0 +1,20 @@
+using Core.DomainModels;
+using Core.IRepositories;
+using Database.DatabaseContext;
+
+namespace Database.Repositories;
+
+public class OrganisationDbRepository : IOrganisationRepository
+{
+    private readonly ApplicationContext _context;
+
+    public OrganisationDbRepository(ApplicationContext context)
+    {
+        _context = context;
+    }
```

Comment:
Primary constructors er supernice og smexy, bruk det der du kan 🤩

```suggestion
public class OrganisationDbRepository(ApplicationContext context) : IOrganisationRepository
{
```

Reply:
🤯